### DISCUSSION TITLE
Updating PowerShell to version 7.* in Windows 10. How to?

### DISCUSSION THREAD

**Discussion by PaulStSmith** [PowerShell/PowerShell#20621] [Q&A] [answered]
I just finishing using the MSI to try to force the installation of the latest version.
The setup wizard said that everything was done and ok.
So, I was **very** disappointed when I checked the `$PSVersionTable`

![image](https://github.com/PowerShell/PowerShell/assets/118175402/7d978152-5035-42f3-84df-40fc8b93d1eb)

I was expecting to see the `PSVersion` entry something like `7.*`

How can I truly upgrade PowerShell?

**Comment by SteveL-MSFT** [comment, marked as answer]
PS7 starts via `pwsh.exe`, not `powershell.exe` as 7 and 5.1 are designed to work side-by-side.  Run `pwsh` then `$psversiontable`.

**Comment by PaulStSmith** [comment]
Thanks. I didn't know.
